src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -60,6 +60,24 @@
             .withEmail(VALID_EMAIL_BOB).withAddress(VALID_ADDRESS_BOB).withUserType(VALID_USERTYPE_BUYER)
             .withPreference(VALID_PREFERENCE_BOB).build();
 
+    // Sellers with properties
+    public static final Person PROTY = new PersonBuilder().withName("Proty Per").withPhone("35748962")
+            .withEmail("hastwo@property.com").withAddress("orchard").withUserType("seller")
```

Comment:
Maybe change the address to a more likely one like "311, Clementi Ave 2, #02-25" (with street name, unit no).

Reply:
Changes made!